eox_tagging/test/test_models.py

```diff
@@ -240,16 +240,17 @@ def test_tag_inmutable(self):
 
     def test_find_by_owner(self):
         """ Used to confirm that can retrieve tags by owner_object."""
-        tags_owned = Tag.objects.find_by_owner(self.owner_object)
-        tags_owned_fake = Tag.objects.find_by_owner(self.fake_owner_object)
+        tags_owned = Tag.objects.find_by_owner(owner_type="user", owner_id={"username": "User"})
```

Comment:
what should happen if owner_type is "CourseOverview" ?

Reply:
Owner in configurations are defined as a user or site (site as default), so it should raise a ValidationError

Reply:
could you add that test

Reply:
yes!

Reply:
https://github.com/eduNEXT/eox-tagging/blob/dccef24197b23772f49e2dbd6c594889b805db3f/eox_tagging/test/test_models.py#L235